contracts/script/deploy/certverifier/README.md

```diff
@@ -23,7 +20,7 @@ To set up the deployment, a config json should be placed in the `config/` folder
 }
 ```
 
-Two sample configs are provided in the `config/` folder for preprod and testnet environments.
+Three sample configs are provided in the `config/` folder for preprod and testnet environments on holesky as well as testnet environment on sepolia.
```

Comment:
IMO the number of configs should not be specified here since it is a double bookkeeping

Reply:
Would prefer to just leave as is if possible